### DISCUSSION TITLE
Have you tested compatibility with Windows 11 and Google Chrome 95?

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [DNSCrypt/dnscrypt-proxy#1924] [Q&A]
Hi
I really like dnscrypt-proxy
I used to use dnscrypt-proxy program on **Windows 10** and I had **no problems**
However, I have **not been able** to use dnscrypt-proxy since I installed **Windows 11** and by the way Google Chrome released version 95.
**I can not use** dnscrypt-proxy in any way in **Windows 11** operating system
please guide me

**Comment by MojtabaFaal** [comment]
Do we have to install anything else **before** using dnscrypt-proxy ?
For example :

- Visual C++ Redistributable ?
- .NET Framework ?
- Java SE Runtime Environment (JRE) ?
- enable virtual machine (VM) ?
- enable Virtualization (VT) ?
- A special generation of Cpu ?
- Free RAM ?
- Install Python ?
- ... ?

**Reply by jedisct1** [reply]
No.
